src/main/java/emissary/output/filter/AbstractFilter.java

```diff
@@ -474,6 +475,20 @@ protected Set<String> getTypesToCheckForNamedView(final IBaseDataObject d, final
         return checkTypes;
     }
 
+    protected boolean denyListContains(final String viewName) {
+        if (this.denylist.contains(viewName))
+            return true;
+        for (String entry : this.denylist) {
+            if (entry.endsWith("*")) { // Wildcard char
+                String base = entry.substring(0, entry.length() - 1);
+                Pattern pattern = Pattern.compile(base + "[a-zA-Z0-9]*");
+                if (pattern.matcher(viewName).find())
+                    return true;
+            }
+        }
```

Comment:
We couldn't want to reconstruct/recompile the Patterns for every processed IBDO.  We'd want to compile them at configuration time and add them to a new `Set<Pattern>`, since the Patterns themselves are data-independent.

Alternatively, we wouldn't use Patterns at all; we'd use `viewName.startsWith(base)`, but even then, we'd want to build the "wildcardDenyList" at configuration time.